### DISCUSSION TITLE
Docker compose - array to string conversion error

### DISCUSSION THREAD

**Discussion by bohtho** [coollabsio/coolify#1506] [Q&A]
What could the reason be for getting an "array to string conversion" error when loading a pretty straightforward docker compose yml-file in a GitHub repository with the docker compose buildpack in beta 149/150 ?

**Comment by andrasbacsai** [comment]
> What could the reason be for getting an "array to string conversion" error when loading a pretty straightforward docker compose yml-file in a GitHub repository with the docker compose buildpack in beta 149/150 ?

Can you please share the yml file as an example?

**Reply by bohtho** [reply]
https://gist.github.com/bohtho/1f258a8f3c638b7f0ad90a16da7f49f0

**Reply by bohtho** [reply]
This is the output in the debug log:

> OUTPUT: Cloning into '/artifacts/wow0swo'...
> [2023-Nov-30 21:37:45.651126] Oops something is not okay, are you okay? 😢
> [2023-Nov-30 21:37:45.667394] Array to string conversion`

**Reply by bohtho** [reply]
Moving/Removing referenced environment properties from the .env file to docker-compose.yml seemed to fix it (even though I also had entered those in Environment Variables in Coolify).

But I also had to reduce "replicas:" in the docker-compose.yml to 1, because else I got an error from Docker that containers cant have identical names. Seems Coolify overwrites where docker would give the replicas slightly different names.

And I had to turn off "Force https" under Advanced in Coolify.
